pep-0655.rst

```diff
@@ -55,12 +55,25 @@ a mix of both required and potentially-missing keys:
        title: str
        year: NotRequired[int]
 
+This PEP also makes it possible to define TypedDicts in the
+:pep:`alternative functional syntax <589#alternative-syntax>`
+with a mix of required and potentially-missing keys,
+which is not currently possible at all because the alternative syntax does
+not support inheritance:
+
+::
```

Comment:
```suggestion
not support inheritance::
```

You can just do this, FYI

Reply:
Aye. I prefer keeping the code block element independent of the preceding paragraph element.

Such separation is also consistent with Pandoc's output, which I used to generate most of the RST syntax originally (from Markdown).